DFC.ServiceTaxonomy.Editor/appsettings.json

```diff
@@ -122,6 +122,6 @@
         "ContainerName": "appregistry",
         "ContainerDraftName": "appregistry-draft",
         "ConnectionString": "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw=="
-    }
+    },
+    "RelatedContentItemIndexTypes": [ "JobProfile", "Page" ]
```

Comment:
does this need to changes based on our new list of content types that we would like to keep https://skillsfundingagency.atlassian.net/wiki/spaces/DFC/pages/4607705146/RelatedContentItemIndex+-+Investigation?